src/pages/restaurant/Restaurant.js

```diff
@@ -93,7 +75,8 @@ class Restaurant extends React.Component {
     return <Menu dishes={dishes} />;
   }
   render() {
-    const { isLoading, errorMessage } = this.state;
+    console.log(this.props);
+    const { isLoading, errorMessage } = this.props;
```

Comment:
the state isLoading and errorMessage could be in present in state only instead of redux store because its only used by one component. but this is also fine i think